### PR TITLE
OSSM-6713 [DOC] Installation and upgrade tasks for OSSM 2.6 release

### PR DESCRIPTION
**Pull request by gwynnemonahan**
**OSSM 2.6**

[OSSM-6713](https://issues.redhat.com//browse/OSSM-6713) [DOC] Installation and upgrade tasks for OSSM 2.6 release

Version(s):
4.12+

Issue:
https://issues.redhat.com/browse/OSSM-6713

Link to docs preview:
https://78142--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/upgrading-ossm.html#ossm-upgrade-25-26-changes_ossm-upgrade

Reviews:
- [x] PM has approved this change.
- [x] Dev has approved this change. 
- [x] QE has approved this change.
- [x] OCP Doc Peer Review

Additional information:
Reviews and dates added for quick reference of what PRs are where for OSSM 2.6.

### CODE REVIEW

File: modules/ossm-upgrade-25-26-changes.adoc [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+== Envoy sidecar container default setting change
+
+To enhance pod startup times, Istio now includes a `startupProbe` in sidecar containers by default. The pod's readiness probes will not start until the Envoy sidecar has started.
+
+//Removed pending verification that SMCP supports disabling startupProbe.
+//You can disable the startup probe:
+
+//[source,yaml]
+//----
+//readinessInitialDelaySeconds: 1
```
Comment (longmuir) [inline modules/ossm-upgrade-25-26-changes.adoc:26]: I asked about this in Slack before, and never got a response, but I'm not sure where this configuration would be set in the SMCP? Do we support these configuration settings or would they need to be set via meshConfig (and thus fall under the techPreview field)?

File: modules/ossm-upgrade-25-26-changes.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+= Upgrade changes from version 2.5 to version 2.6
+
+//Jaeger disabled by default goes in "Upgrading" and 2.6 Rel Notes
+== {JaegerName} default setting change
+
+This release disables {JaegerName} by default for new instances of the `ServiceMeshControlPlane` resource.
+
+When updating existing instances of the `ServiceMeshControlPlane` resource to {SMProductName} version 2.6, {JaegerShortName} remains enabled by default.
+
+{SMProductName} 2.6 is the last release that includes support for {JaegerName} and {es-op}. Both {JaegerShortNamee} and {es-op} will be removed in the next release. If you are currently using {JaegerShortName} and {es-op}, you need to switch to {TempoName} and {OTELName}.
```
Comment (abrennan89) [inline modules/ossm-upgrade-25-26-changes.adoc:15]:
```suggestion
{SMProductName} 2.6 is the last release that includes support for {JaegerName} and {es-op}. Both {JaegerShortName} and {es-op} will be removed in the next release. If you are currently using {JaegerShortName} and {es-op}, you must migrate to {TempoName} and {OTELName}.
```
Suggestion for minimalism / wording, also typo in attribute name

File: modules/ossm-upgrade-25-26-changes.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+== Envoy sidecar container default setting change
+
+To enhance pod startup times, Istio now includes a `startupProbe` in sidecar containers by default. The pod's readiness probes will not start until the Envoy sidecar has started.
```
Comment (abrennan89) [inline modules/ossm-upgrade-25-26-changes.adoc:19]:
```suggestion
To enhance pod startup times, Istio now includes a `startupProbe` in sidecar containers by default. The pod's readiness probes do not start until the Envoy sidecar has started.
```
present tense

**Review by abrennan89 — COMMENTED** [review]
Couple of minor things

**Comment by gwynnemonahan** [comment]
Rebase 08/06/2024

**Comment by gwynnemonahan** [comment]
Manual test on 4.12, 08/06/2024. No conflicts.

**Comment by gwynnemonahan** [comment]
Rebased against "main" 08/07/2024.

**Review by adellape — COMMENTED** [review]
Not blocking merge, but a note.

**Pull request merged**